### ISSUE TITLE
role=button requires space to trigger action (accessibility)

### ISSUE THREAD

**Issue by masi** [nolimits4web/swiper#3595]
This is a:
* [x] bug
* [x] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.3.8

### What you did
Enabled modules Pagination and Acessibility

### Expected Behavior
Bullets can be triggered by space key.

### Actual Behavior
Bullets cannot be triggered by space key (only by enter key).

### Related Documents:
https://www.w3.org/TR/wai-aria-practices/#button
https://www.w3.org/TR/wai-aria-practices/examples/button/button.html

**Comment by masi** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Well?

**Comment by dsuriano** [comment]
**+1** for a resolution of this issue. According to [WebAIM](https://webaim.org/techniques/keyboard/), buttons should be activated using either `Enter` or `Spacebar`. `Enter` works on both pagination and arrows, but `Spacebar` doesn't. Likely just missing a simple key-code condition check in the `onKeyPress` handler, but I haven't looked into the code myself.

![image](https://user-images.githubusercontent.com/633004/108760219-3d136e00-7502-11eb-93f6-a0dafc174945.png)
